### PR TITLE
[FIX] replace DefaultFormatBundle/3D with Pack(3D)DetInputs

### PR DESCRIPTION
**Pull request by cxiang26**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: configs/_base_/datasets/nuim-instance.py [hunk 1]
```diff
@@ -40,8 +39,10 @@
             dict(type='Normalize', **img_norm_cfg),
```
Comment (ZCMax) [inline configs/_base_/datasets/nuim-instance.py:39]: we need to remove `Normalize`, `Pad` and `ImageToTensor` in pipeline which has been done in `data_preprocessor` in 1.1 version

File: configs/nuimages/htc_r50_fpn_1x_nuim.py [hunk 2]
```diff
@@ -33,10 +33,7 @@
     dict(type='Normalize', **img_norm_cfg),
```
Comment (ZCMax) [inline configs/nuimages/htc_r50_fpn_1x_nuim.py:33]: also same here

File: configs/nuimages/mask-rcnn_r50_caffe_fpn_1x_nuim.py [hunk 3]
```diff
@@ -22,8 +22,7 @@
     dict(type='RandomFlip', flip_ratio=0.5),
     dict(type='Normalize', **img_norm_cfg),
     dict(type='Pad', size_divisor=32),
```
Comment (ZCMax) [inline configs/nuimages/mask-rcnn_r50_caffe_fpn_1x_nuim.py:24]: same here

**Comment by ZCMax** [comment]
Thanks for your PR,  please fix the comments ~

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**